app/views/roles/_new_modal.html.erb

```diff
@@ -13,25 +13,29 @@
 				<div class='field'>
 					<div class='field-inline'>
 						<input type="radio" name="role[name]" value="nonprofit_admin" id='roleName--admin'>
-						<label for='roleName--admin'><strong>Nonprofit Admin</strong></label>
+						<label for='roleName--admin'><strong><%= t('roles.new.nonprofit_admin') %></strong></label>
 					</div>
-					<p>Full permissions, including bank payouts</p>
+					<p><%= t('roles.new.full_permissions_including_bank_payouts') %></p>
 				</div>
 
 				<div class='field'>
 					<div class='field-inline'>
 						<input type="radio" name="role[name]" value="nonprofit_associate" checked='checked' id='roleName--assoc'>
-						<label for='roleName--assoc'><strong>Nonprofit Associate</strong></label>
+						<label for='roleName--assoc'><strong><%= t('roles.new.nonprofit_associate') %></strong></label>
 					</div>
-					<p>Edit content and view records</p>
+					<p><%= t('roles.new.edit_content_and_view_records') %></p>
 				</div>
 
-				<p><strong>Important!<br/> Double check this email address</strong></p>
-				<input name='role[email]' type='email' placeholder='Email' id='role-email-field' required><br>
-				<input name='role[email_confirmation]' type='email' placeholder='Confirm email' id='role-email-field' required parsley-equalto='#role-email-field'>
+				<p><strong>
+					<%= t('roles.new.important') %>
+					<br>
+					<%= t('roles.new.double_check_this_email_address') %>
+				</strong></p>
```

Comment:
Is there any language that we can foresee where the order of these two UI elements would be reversed?

To be clear, we don't need to evaluate every language that exists, I'd just like to avoid a situation where we have 10 languages and then we realize we have to change all of them because a new language we add expects it in the opposite order.

Reply:
I think it would be the case if it was a single sentence instead of two, but I don't know if there's any language that would make that different. It behaves like two different HTML elements, I think the person chose to use `<br>` instead of two `<p>` elements because it was easier than handling the spacing.

Reply:
Hmmm, I think we should go with two elements and then work through the problem later if it exists.